demisto_sdk/commands/update_release_notes/tests/update_rn_test.py

```diff
@@ -1128,6 +1128,42 @@ def test_add_and_modify_files_without_update_docker_image(self, mocker):
         os.remove('demisto_sdk/commands/update_release_notes/tests_data/Packs/release_notes/1_1_0.md')
         assert 'Updated the Docker image to: *dockerimage:python/test:1243*' not in RN
 
+    def test_new_integration_docker_not_updated(self, mocker):
+        """
+        Given
+            - New integration created.
```

Comment:
Also pack exist right?

Reply:
doesn't really matter if pack existed or not